### PR TITLE
made twitter iframes render a bit nicer on small devices

### PR DESCRIPTION
**Pull request by bookercodes**
This pull request should make the Twitter iframes a bit more legible on smaller devices. 

Note that I removed the utlity classes as they make updating styles from within a media query awkward.

Also note that I added a custom media query, as I felt the breakpoint was unique. 

In a future pull request, I will utilize PostCSS to use vars, mixins, etc. to reduce duplication and improve maintainability, so please don't worry too much about that for this pull request.

Thanks!

### CODE REVIEW

**Comment by bookercodes** [comment]
One more thing: Regrettably, I had to use `!important` to override the Twitter iframe styles, which are defined in-line by the API.

File: styles.css [hunk 1]
```diff
@@ -96,6 +96,22 @@ h3 small {
 }
 
 
+.twitter-timeline-rendered {
+  margin: auto;
+  margin-bottom: 25px !important;
```
Comment (kentcdodds) [inline styles.css:101]: no worries. Sometimes you have to do this when dealing with third party stuff. That's life :-)

**Comment by kentcdodds** [comment]
Ah, sorry, I used [rebase and merge](https://github.com/kentcdodds/rebase-and-merge) to merge this PR and if the branch itself hasn't been rebased to master then the commits are rewritten and github doesn't recognize this PR as being merged. But it has! Here's [the commit](https://github.com/javascriptair/site/commit/bfd43305449fd9afe8a98d2c7b75b3a1987503d4) (you're still the author) :-) Cheers!

**Comment by bookercodes** [comment]
@kentcdodds No worries and thank you for merging.

**Pull request closed without merging**